lib/lint-messages.js

```diff
@@ -57,11 +57,21 @@ function lintFileContents (messages, js) {
     lineNumber++
     const parsedImport = parseImportLine(line)
     if (parsedImport) {
-      [importedAtRootAs, importedMembersAs] = parsedImport
+      ;[importedAtRootAs, importedMembersAs] = parsedImport
```

Comment:
That's the standard use case for advocating pro-semicolons. For example:

```js
const a = b
[0].map(doSomething)
```
Would be interpreted as
```js
const a = b[0].map(doSomething)
```